docs/tutorials/basic/symbol.md

```diff
@@ -217,24 +379,28 @@ c.tojson() == c2.tojson()
 
 ## Customized Symbol
 
-Most operators such as `mx.sym.Convolution` and `mx.sym.Reshape` are implemented
-in C++ for better performance. MXNet also allows users to write new operators
-using any frontend language such as Python. It often makes the developing and
-debugging much easier.
-
-To implement an operator in Python, we just need to define the two computation
-methods `forward` and `backward` with several methods for querying the
-properties, such as `list_arguments` and `infer_shape`.
-
-`NDArray` is the default type of arguments in both `forward` and
-`backward`. Therefore we often also implement the computation with `NDArray`
-operations. To show the flexibility of MXNet, however, we will demonstrate an
-implementation of the `softmax` layer using NumPy. Though a NumPy based operator
-can be only run on CPU and also lose some optimizations which can be applied on
-NDArray, it enjoys the rich functionalities provided by NumPy.
-
-We first create a subclass of `mx.operator.CustomOp` and then define `forward`
-and `backward`.
+Most operators such as `mx.sym.Convolution` and `mx.sym.Reshape`
```

Comment:
are we duplicating the new operator guide here? Consider putting a link instead?